.circleci/config.yml.in

```diff
@@ -809,14 +809,6 @@ workflows:
   build:
     jobs:
       - circleci_consistency
```

Comment:
Can we move circleci_concistency to unit test workflow? because that way we can completely get rid of build workflow in CCI.

Reply:
done!